scripts/quests/otherAreas/Elderly_Pursuits.lua

```diff
@@ -0,0 +1,183 @@
+                player:getQuestStatus(xi.quest.log_id.OTHER_AREAS, xi.quest.id.otherAreas.SECRETS_OF_OVENS_LOST) == QUEST_COMPLETED
+        end,
+
+        [xi.zone.TAVNAZIAN_SAFEHOLD] =
+        {
+            ['Despachiaire'] =
+            {
+                onTrigger = function(player, npc)
+                    return quest:progressEvent(517)
+                end,
+            },
+
+            onEventFinish =
+            {
+                [517] = function(player, csid, option, npc)
+                    quest:begin(player) -- begin quest
+                    quest:setVar(player, 'Prog', 1)
+                    return quest:keyItem(xi.ki.ANTIQUE_AMULET)
+                end,
+            },
+        },
+    },
+
+    {
+        check = function(player, status, vars)
+            return status == QUEST_ACCEPTED
+        end,
+
+        [xi.zone.TAVNAZIAN_SAFEHOLD] =
+        {
+            ['Despachiaire'] =
+            {
+                onTrigger = function(player, npc)
+                    if quest:getVar(player, 'Prog') == 1 then
+                        return quest:progressEvent(318)
+                    elseif quest:getVar(player, 'Prog') == 4 then
+                        return quest:progressEvent(518)
+                    end
+                end,
+            },
+
+            onEventFinish =
+            {
+                [518] = function(player, csid, option, npc)
+                    player:delKeyItem(xi.ki.CATHEDRAL_MEDALLION)
+                    quest:complete(player)
+                end,
+            },
+        },
+
+        [xi.zone.SOUTHERN_SAN_DORIA] =
+        {
+            ['Rouva'] =
+            {
+                onTrigger = function(player, npc)
+                    if
+                        quest:getVar(player, 'Prog') == 1 and
+                        player:hasKeyItem(xi.ki.ANTIQUE_AMULET)
+                    then
+                        return quest:progressEvent(747)
+                    elseif
+                        quest:getVar(player, 'Prog') == 3 and
+                        player:hasKeyItem(xi.ki.CATHEDRAL_MEDALLION)
+                    then
+                        return quest:progressEvent(748)
+                    end
+                end,
+            },
+
+            onEventFinish =
+            {
+                [747] = function(player, csid, option, npc)
+                    quest:setVar(player, 'Prog', 2)
+                end,
+
+                [748] = function(player, csid, option, npc)
+                    quest:setVar(player, 'Prog', 4)
+                end,
+            },
+        },
+
+        [xi.zone.CARPENTERS_LANDING] =
+        {
+            ['qm_para'] =
+            {
+                onTrigger = function(player, npc)
+                    if quest:getVar(player, 'Prog') == 2 then
+                        if quest:getVar(player, 'ParaKilled') == 1 then
+                            player:messageSpecial(carpentersLandingID.text.POLISH, xi.ki.ANTIQUE_AMULET)
```

Comment:
Does this messageSpecial replace the generic Obtained: <keyitem> or are both displayed?  The returned object will display the latter

Reply:
Both are displayed, the POLISH text and Obtained text were both shown from my testing - the reason I added the return was because the default text was displaying without it.

Capture:
<img width="326" alt="Screenshot 2024-02-27 at 12 15 12 PM" src="https://github.com/LandSandBoat/server/assets/33179100/f550f737-c36b-421e-a42d-0448638898ed">
Local:
<img width="551" alt="Screenshot 2024-02-27 at 12 59 44 PM" src="https://github.com/LandSandBoat/server/assets/33179100/f987bf47-845a-4388-bacc-f9fbc8d28347">